packages/a11y-base/src/keyboard-direction-mixin.js

```diff
@@ -150,30 +154,27 @@ export const KeyboardDirectionMixin = (superclass) =>
      * Focus the item at given index. Override this method to add custom logic.
      *
      * @param {number} index
+     * @param {FocusOptions} options
      * @param {boolean} navigating
```

Comment:
Unrelated to these changes, but it seems that `navigating` parameter is not being used anywhere. Are we keeping it for backward compatibility or I missed something?

Reply:
Right, I missed that one.